lab-davide/lib/greet.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+module.exports = function(name){
+  return 'hello ' + name +' how are you?';
```

Comment:
This is good, and I won't deduct 1 for adding +' how are you?' (this time), but please make sure to follow the assignment specifications to the letter, so if it says return 'hello ' + name do exactly that.